packages/cli/src/build/bundle.rs

```diff
@@ -354,38 +353,19 @@ impl AppBundle {
         Ok(())
     }
 
-    /// Copy the assets out of the manifest and into the target location
-    ///
-    /// Should be the same on all platforms - just copy over the assets from the manifest into the output directory
-    async fn write_assets(&self) -> Result<()> {
-        // Server doesn't need assets - web will provide them
-        if self.build.build.platform() == Platform::Server {
-            return Ok(());
-        }
-
+    /// Clean up any assets that no longer exist in the manifest
+    async fn clear_old_assets(&self) -> Result<()> {
         let asset_dir = self.build.asset_dir();
 
         // First, clear the asset dir of any files that don't exist in the new manifest
         _ = tokio::fs::create_dir_all(&asset_dir).await;
         // Create a set of all the paths that new files will be bundled to
-        let mut keep_bundled_output_paths: HashSet<_> = self
+        let keep_bundled_output_paths: HashSet<_> = self
             .app
             .assets
-            .assets
-            .values()
+            .assets()
             .map(|a| asset_dir.join(a.bundled_path()))
             .collect();
-        // The CLI creates a .version file in the asset dir to keep track of what version of the optimizer
```

Comment:
The optimizer's version is now included in the file hash so we don't need to keep track of an asset optimizer version file